api/src/pcapi/serialization/decorator.py

```diff
@@ -153,13 +153,7 @@ def sync_validate(*args: Any, **kwargs: Any) -> Response:
             if query_in_kwargs:
                 kwargs["query"] = query_in_kwargs(**query_params)
             if form_in_kwargs:
-                try:
-                    kwargs["form"] = form_in_kwargs(**form)
-                except pydantic.ValidationError as validation_errors:
-                    error_dict = {}
-                    for errors in validation_errors.errors():
-                        error_dict[errors["loc"][0]] = errors["msg"]
-                    raise ApiErrors(error_dict)
+                kwargs["form"] = form_in_kwargs(**form)
```

Comment:
Plus de try/except _because_ : _spectree now handles form in requests_ ?

Reply:
Yep !